### ISSUE TITLE
Support for issues

### ISSUE THREAD

**Issue by SirWindfield** [actions/labeler#17]
Something similar for issues would be fantastic. Maybe instead of a file glob a regex that gets matched against the description and title (might be configurable later on). I guess it could live under the same repo. If not, maybe a new one like `issue-labeler` would be good to have.

Not sure if you want to tackle this one down. I can give it a try though. Is there some easy way to test actions besides pushing them, creating a test repo and testing them there out? 

And is there a way to "apply" for the actions organization once an action is ready for production? I just think that this one would really fit into the core actions you provide!

**Comment by damccorm** [comment]
I'm not on the actions team at this point, but https://github.com/damccorm/tag-ur-it is an action that does labeling based on issue content if you want to check it out.

**Comment by jenlampton** [comment]
I just checked out damccorm/tag-ur-it but it looks like that adds labels to issues based on text in the issue. I'm looking for a solution that will add labels (specifically `has pull request` and `needs code review`) to issues when a PR is filed.

Has anyone seen (or created) something that does this?

**Comment by damccorm** [comment]
@jenlampton I think you can do this with issue templates. For example, check out this template which does this with its header - https://raw.githubusercontent.com/github/hub/master/.github/ISSUE_TEMPLATE/bug_report.md

specifically, the header:

```
---
name: Bug report
about: Unexpected or broken behavior of "hub" command-line tool
title: ''
labels: bug
assignees: ''

---
```

automatically adds the bug label any time an issue is opened. You can get this formatting automatically by going through the issue template setup - https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/configuring-issue-templates-for-your-repository

**Comment by jenlampton** [comment]
Yes, we already have labels added to our issues *when they are created* by using the templates. I am looking to have new labels added to issues *when a pull request is filed*, and *when a pull request fails automated tests*

So far I have only found solutions for labels on issues when something happens to the issue, and labels on PRs when something happens to the PR. But nothing that will label an issue based on what happens to it's PR.
